src/sass/components/govcy-header.scss

```diff
@@ -44,6 +46,14 @@ $govcy-sub-menu-color:#577589;
 			~ {
```

Comment:
From `.menu-btn` down bootstrap classes are globally overwritten because of the closing curly brace on line 39 closing the `.govcy-header` class

Reply:
![Uploading Screenshot 2021-12-13 at 09.40.12.png…]()

Reply:
![Screenshot 2021-12-13 at 09 40 12](https://user-images.githubusercontent.com/3469840/145788359-0bc3ce58-41b0-4ae6-b954-5216ed46ad54.png)

Reply:
yea it will be fixed

Reply:
👍